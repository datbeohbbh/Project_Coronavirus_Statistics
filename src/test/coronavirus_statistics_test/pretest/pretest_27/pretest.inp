33 30
? 5
q 1 28 486
q 1 27 437
q 1 3 125
q 1 22 87
q 1 23 118
? 4 4
q 1 25 313
? 4 1
q 1 26 384
? 4 4
? 4 4
q 1 4 200
q 1 19 36
? 6
q 2 23 58
q 1 18 141
q 1 1 16
q 1 33 208
? 7
q 1 21 372
q 3 27 327
? 6
? 6
? 4 2
? 6
? 6
? 7
? 4 11
? 7