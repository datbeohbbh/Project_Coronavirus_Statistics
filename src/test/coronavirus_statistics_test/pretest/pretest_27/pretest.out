Query ? 5 :
The number of safe cities is 33

Query ? 4 4 :
The answer is the city with the index 27 
- Information of the city 27 -
-> Number of current patients : 437
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 22 
- Information of the city 22 -
-> Number of current patients : 87
-> Has recovered : 0
-> Was dead : 0

Query ? 4 4 :
The answer is the city with the index 25 
- Information of the city 25 -
-> Number of current patients : 313
-> Has recovered : 0
-> Was dead : 0

Query ? 4 4 :
The answer is the city with the index 25 
- Information of the city 25 -
-> Number of current patients : 313
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
Currently, no recovery cases have been recorded

Query ? 7 :
Currently, no deaths have been recorded

Query ? 6 :
The city with the highest recovery is city with the index 23
Quantity: 58 people

Query ? 6 :
The city with the highest recovery is city with the index 23
Quantity: 58 people

Query ? 4 2 :
The answer is the city with the index 19 
- Information of the city 19 -
-> Number of current patients : 36
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 23
Quantity: 58 people

Query ? 6 :
The city with the highest recovery is city with the index 23
Quantity: 58 people

Query ? 7 :
The city with the highest death is city with the index 27
Quantity: 327 people

Query ? 4 11 :
The answer is the city with the index 21 
- Information of the city 21 -
-> Number of current patients : 372
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 27
Quantity: 327 people

